### PR TITLE
Add a special response to support the akhat. LONG LIVE AKHAT!

### PR DESCRIPTION
**Pull request by Sagghaa**
Akhat forever

### CODE REVIEW

File: src/serverCommands.ts [hunk 1]
```diff
@@ -41,19 +42,21 @@ export const handleMessage = async (message: Discord.Message, handler: ServerHan
 			}
 			break;
 		case "mute":
-			if (!message.member.roles.find((r: Discord.Role) => r.name == "Admin")) {
+			if (!message.member.roles.find((r: Discord.Role) => r.name == "Admin") && !message.member.roles.find((r: Discord.Role) => r.name == "Akhat")) {
```
Comment (OctoMeep) [inline src/serverCommands.ts]: nice try

File: src/serverCommands.ts [hunk 2]
```diff
@@ -79,10 +82,11 @@ export const handleMessage = async (message: Discord.Message, handler: ServerHan
 			}
 			break;
 		case "strike":
-			if (!message.member.roles.find((r: Discord.Role) => r.name == "Admin")) {
```
Comment (OctoMeep) [inline src/serverCommands.ts]: also nice try

File: src/ServerHandler.ts [hunk 3]
```diff
@@ -215,6 +225,10 @@ export default class ServerHandler {
 
 	}
 
+	
+
+	
+
```
Comment (OctoMeep) [inline src/ServerHandler.ts]: why

File: src/ServerHandler.ts [hunk 4]
```diff
@@ -25,11 +25,19 @@ export default class ServerHandler {
...
 		if (message.author.bot) return;
 		if (!this.initialized) await this.initialize(false, message);
 		if (!this.active) return;
+		if(message.content === "Akhat above all" || message.content === akhat.toString()){
+			
+			await message.channel.send(`${akhat} I only support the Akhat, everyone else is inferior.`);
+			return;
+		}
+		if(message.content === "comrades")
+		await message.channel.send("Comrades are gay");
```
Comment (OctoMeep) [inline src/ServerHandler.ts]: please remove

Comment (Sagghaa) [reply src/ServerHandler.ts]: this is actually reasonable

Comment (OctoMeep) [reply src/ServerHandler.ts]: no

File: src/serverCommands.ts [hunk 5]
```diff
@@ -41,19 +42,21 @@ export const handleMessage = async (message: Discord.Message, handler: ServerHan
 			}
 			break;
 		case "mute":
-			if (!message.member.roles.find((r: Discord.Role) => r.name == "Admin")) {
+			if (!message.member.roles.find((r: Discord.Role) => r.name == "Admin"))) {
```
Comment (OctoMeep) [inline src/serverCommands.ts]:
```suggestion
			if (!message.member.roles.find((r: Discord.Role) => r.name == "Admin")) {
```

File: src/serverCommands.ts [hunk 6]
```diff
@@ -79,10 +82,11 @@ export const handleMessage = async (message: Discord.Message, handler: ServerHan
 			}
 			break;
 		case "strike":
-			if (!message.member.roles.find((r: Discord.Role) => r.name == "Admin")) {
+			if (!message.member.roles.find((r: Discord.Role) => r.name == "Admin"))) {
```
Comment (OctoMeep) [inline src/serverCommands.ts]:
```suggestion
			if (!message.member.roles.find((r: Discord.Role) => r.name == "Admin")) {
```

**Pull request merged**